GVFS/GVFS.Service.UI/GVFSToastRequestHandler.cs

```diff
@@ -0,0 +1,137 @@
+﻿using GVFS.Common;
+using GVFS.Common.NamedPipes;
+using GVFS.Common.Tracing;
+using System;
+using System.Diagnostics;
+
+namespace GVFS.Service.UI
+{
+    public class GVFSToastRequestHandler
+    {
+        private const string GVFSAutomountStartTitle= "GVFS Automount";

```

Comment:
Shouldn't these be named "VFS ..."?

```suggestion
        private const string GVFSAutomountStartTitle= "VFS Automount";
```

Reply:
Same applies to the rest of the strings here (and the property names?).

Reply:
Updated the message text and properties in this file. (But there are so many properties, classes and files that use gvfs in their names today. )